packages/utils/src/get-all-content-with-refs.ts

```diff
@@ -0,0 +1,32 @@
+import { BuilderContent, BuilderElement, Builder } from '@builder.io/sdk';
+import traverse from 'traverse';
+
+export const isBuilderElement = (item: unknown): item is BuilderElement => {
+  return Boolean((item as any)?.['@type'] === '@builder.io/sdk:Element');
+};
+
+async function resolveRefs(content: any, apiKey: string, builderInstance: Builder) {
+  const promises: Promise<void>[] = [];
+  const result = traverse(content).map(function (child) {
+    if (child && child['@type'] === '@builder.io/core:Reference') {
+      if (child.model) {
+        promises.push(
+          builderInstance.getAll(child.model, { query: { id: child.id } }).then(async value => {
```

Comment:
this should be `builderInstance.get(child.model, { options: { query: {id : child.id} }`, ref.value should be a content object